packages/slate/src/interfaces/text.ts

```diff
@@ -37,6 +36,12 @@ export const Text: TextInterface = {
   ): boolean {
     const { loose = false } = options
 
+    function omit(obj: Record<any, any>, key: string) {
+      const { [key]: _, ...rest } = obj
```

Comment:
`omit` is only used one way, with `text` as the key, so perhaps we should just make it specific for this case?

Reply:
you're right, addressed in the lates commit